.github/workflows/release.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          asset_paths: '["./kyuubi-*tar.gz"]'
+          asset_paths: '["./kyuubi-*tgz"]'
       - name: Update Release Note
```

Comment:
does this action actually be used？

Reply:
I'm going to remove this job. There are still lots of works to do for apache release.

Reply:
thank you